### PR TITLE
implement feature flags

### LINKED ISSUE #43: Feature flags for capabilities
## Overview
As a developer, I should be able to enable or disable each capability in a simple manner, preferably from a single place in code. If certain feature starts misbehaving, we can temporary disable it until the issue is solved, without jeopardizing stability of the whole application.

### LINKED ISSUE #48: Highlight active menu item
## Overview
As a user, I should have a menu indicator for the page I currently have open.

### PR DESCRIPTION
**Pull request by linhnk-ips**

### CODE REVIEW

File: src/pwa/lib/featureFlags.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export const featureFlags = () => {
+    return {
+        isEnableNotification: true,
+        isEnableWebAuth: true
```
Comment (uveta) [inline src/pwa/lib/featureFlags.ts]: Not all features are covered. I would include and utilize following:
- isEnableCalling
- isEnableCamera
- isEnableLocationServices

Comment (linhnk-ips) [reply src/pwa/lib/featureFlags.ts]: I updated. when feature is disable, the page with be redirected to disableFeature page

Comment (uveta) [reply src/pwa/lib/featureFlags.ts]: I would still like to have isEnableNotification and isEnableWebAuth in feature flags.
If isEnableNotification is false, remove component from settings, and try to exclude all Firebase integration.
if isEnableWebAuth is false, remove component from settings, and try to exclude all Web Authentication API integration.

Comment (linhnk-ips) [reply src/pwa/lib/featureFlags.ts]: I bring it back. Please help me take a look at the last commit

**Review by uveta — APPROVED** [review]

**Pull request merged**